keyboards/honeycomb/honeycomb.h

```diff
@@ -0,0 +1,40 @@
+#ifndef honeycomb_H
+#define honeycomb_H
+
+#include "quantum.h"
+#include "matrix.h"
+#include "backlight.h"
+#include <stddef.h>
+
+#define red_led_off   PORTF |= (1<<6)
+#define red_led_on    PORTF &= ~(1<<6)
+#define blu_led_off   PORTF |= (1<<5)
+#define blu_led_on    PORTF &= ~(1<<5)
+#define grn_led_off   PORTD |= (1<<1)
+#define grn_led_on    PORTD &= ~(1<<1)
+
+#define set_led_off     red_led_off; grn_led_off; blu_led_off
```

Comment:
```suggestion
#define set_led_off()     red_led_off(); grn_led_off(); blu_led_off()
```

Reply:
Multiple statements in a macro can be problematic. Turn it into a parenthesized expression instead:
```suggestion
#define SET_LED_OFF()   (RED_LED_OFF(), GRN_LED_OFF(), BLU_LED_OFF())
```